mergin/client.py

```diff
@@ -372,15 +373,26 @@ def push_project(self, directory):
         changes = project_changes(server_info["files"], files)
         count = sum(len(items) for items in changes.values())
         if count:
-            def fields():
-                yield Field("changes", json.dumps(changes).encode("utf-8"))
-                for file in (changes["added"] + changes["updated"]):
-                    path = file["path"]
-                    with open(os.path.join(directory, path), "rb") as f:
-                        yield Field(path, f, filename=path, content_type="application/octet-stream")
-
-            encoder = MultipartEncoder(fields())
-            resp = self.post("/v1/project/data_sync/{}".format(project_path), encoder, encoder.get_headers())
+            # Custom MultipartEncoder doesn't compute Content-Length,
```

Comment:
Is it necessary to have commented-out code on master? Please, rather describe a reason for changes in a PR or give a reference to an issue.

Reply:
I would leave it for time being as we might want to switch back to custom encoder (provided that server issue is resolved).